cmd/genji/shell/command.go

```diff
@@ -52,43 +52,58 @@ func runTablesCmd(db *genji.DB, cmd []string) error {
 
 // displayTableIndex prints all indexes that the given table contains.
 func displayTableIndex(db *genji.DB, tableName string) error {
-	err := db.View(func(tx *genji.Tx) error {
-		t, err := tx.GetTable(tableName)
-		if err != nil {
-			return err
-		}
+	ctx := context.Background()
+	res, err := db.Query(ctx, "SELECT index_name FROM __genji_indexes")
```

Comment:
By exposing the __genji_indexes table, you don't need to use `t.GetIndex` anymore, You know have access to all of these fields: https://github.com/genjidb/genji/blob/9da693665fe93d3fcfdffe3312a48712fc40023f/database/config.go#L377:L388

Reply:
Thank you !